theme/errorList.less

```diff
@@ -1,48 +1,60 @@
 @summary-hover-color: #00000067;
 
+// Expanded error list
 .errorList {
     position: relative;
-    min-height: 200px;
+    max-height: 16em;
+    min-height: 16em;
+    border-top: 2px #e5e5e5 solid;
```

Comment:
probably worth a less variable
I'd recommend a less variable for all colors by default